cockatrice/src/user_context_menu.cpp

```diff
@@ -89,6 +94,27 @@ void UserContextMenu::banUser_processUserInfoResponse(const Response &r)
     dlg->show();
 }
 
+void UserContextMenu::adjustMod_processUserResponse(const Response &resp, const CommandContainer &commandContainer)
```

Comment:
The user menu is were its fired from, why wouldnt the response be processed from it as well?  You quicker than lightning and going to close all your context menus before the server has time to process the command?